sessionctx/variable/tidb_vars.go

```diff
@@ -98,6 +98,9 @@ const (
 
 	// tidb_enable_streaming enables TiDB to use streaming API for coprocessor requests.
 	TiDBEnableStreaming = "tidb_enable_streaming"
+
+	// tidb_optimizer_selectivity_level is used for control the content of each operator's statistic information.
```

Comment:
may be we can simplify the comment to "is used to control the selectivity estimation level"?